site-src/api-types/backendtlspolicy.md

```diff
@@ -0,0 +1,133 @@
+# BackendTLSPolicy
+
+??? example "Experimental Channel in v1.0.0+"
+
+    The `BackendTLSPolicy` resource is Alpha and part of the Experimental Channel in `v1.0.0+`.
+
+!!! info "Experimental Channel"
+
+    The `BackendTLSPolicy` resource described below is currently only included in the "Experimental" channel
+    of Gateway API. For more information on release channels, refer to the [related documentation](https://gateway-api.sigs.k8s.io/concepts/versioning).
+
+
+[BackendTLSPolicy][backendtlspolicy] is a Gateway API type for specifying the TLS configuration
+of the connection from the Gateway to a backend pod/s via the Service API object.
+
+## Background
+
+`BackendTLSPolicy` specifically addresses the configuration of TLS in order to convey HTTPS from the Gateway
+dataplane to the backend.  This is referred to as "backend TLS termination" and enables the Gateway to know
+how to connect to a backend pod that has its own certificate.  
+
+While there are other API objects provided for TLS to be configured for **passthrough** and **edge** termination,
+this API object allows users to specifically configure **backend** TLS termination.  For more information on TLS
+configuration in Gateway API, see [TLS Configuration](/guides/tls/).
+
+![Image showing three Termination Types](/images/tls-termination-types.png)
+
+BackendTLSPolicy is a Direct [PolicyAttachment](/reference/policy-attachment/) without defaults or overrides,
+applied to a Service that accesses a backend, where the BackendTLSPolicy resides in the same namespace as the
+Service to which it is applied. The BackendTLSPolicy and the Service must reside in the same namespace in order
+to prevent the complications involved with sharing trust across namespace boundaries.  
+
+One can reuse the same BackendTLSPolicy for all the different Routes that might point to a referenced Service.
```

Comment:
Does this imply "all Routes [attached to a Gateway with support for BackendTLSPolicy] that point to a referenced Service should/must respect a configured policy"?

I think this could be slightly more clear on whether respecting this configuration is optional ("one can reuse") or expected in scenarios with multiple Routes.

Reply:
Good suggestion @mikemorris, maybe something like this? Once this graduates to standard channel, I think "Should" will become "Must" here.
```suggestion
All Gateway API Routes that point to a referenced Service should respect a configured BackendTLSPolicy.
```

Reply:
Agreed. Thanks!